### DISCUSSION TITLE
Object as value in Controller

### DISCUSSION THREAD

**Discussion by ghost** [react-hook-form/react-hook-form#9942] [General] [answered]
Hello, I would like to know if it is possible to set an object as input value like this => { value: "", id: "", key: ""} and when the value changes only updated the value on value inside the object

**Comment by bluebill1049** [comment, marked as answer]
It's possible, you can always host your own state: https://codesandbox.io/s/usecontroller-own-state-wncet2?file=/src/App.tsx
